backlog.txt

```diff
@@ -8,22 +8,7 @@ Sources might help:
 - https://docs.expo.io/guides/configuring-statusbar/#factoring-the-status-bar-in-with-your
 - Check with React Native ZTM Academy course from Udemy
 
-2ND ISSUE: When logged in or signed up, you will be taken to application. When user closes
-the app in their device and opens it, it crashes. Trying to figure out how the user
-stay logged in when closing the app and opening
-
-Sources might help:
-- https://www.youtube.com/watch?v=-N12hjV3DPQ
-- https://www.youtube.com/watch?v=nQVCkqvU1uE&t=1837s
-- https://www.youtube.com/watch?v=gvF6sFIPfsQ
-- https://github.com/ReactNativeSchool/getting-started-react-navigation-v5/blob/master/App/index.js
-- https://github.com/italoiz/react-navigation-reset-bottom-tabs-logout
-- https://reactnavigation.org/docs/auth-flow/
-- https://github.com/bithoven-dev/ReactNavigationAuthenticationFlowsWithHooks/blob/140f4b0b78/src/hooks/useAuth.js
-- https://github.com/itzpradip/react-navigation-v5-mix/blob/master/App.js
-- https://dev.to/embeddednature/create-an-authorization-flow-with-react-navigation-5-x-2pkh
-
-3RD ISSUE: Try to figure out on how to handle keyboard when focus on textinput in Post screen
+2ND ISSUE: Try to figure out on how to handle keyboard when focus on textinput in Post screen
```

Comment:
I think you can either put your docs within backlog via Trello or make a report that ticket on Issue.

Reply:
+1 to report issues using Github instead of keeping them in a text file here. 

https://github.com/AustinVaday/SFH-App/issues